engine/runtime-compiler/src/main/java/org/enso/compiler/pass/analyse/types/BuiltinTypes.java

```diff
@@ -4,31 +4,47 @@
 import org.enso.pkg.QualifiedName$;
 
 /** A helper class providing the builtin types. */
-public class BuiltinTypes {
-  // TODO in next iterations we will want to resolve descriptions of these types based on the loaded
-  // std-lib (from PackageRepository, if available). Note that if the std-lib is not imported,
-  // some builtin types have different names - this should be handled here in some sane way.
+public final class BuiltinTypes {
+  private BuiltinTypes() {}
 
-  public final TypeRepresentation INTEGER = fromQualifiedName("Standard.Base.Data.Numbers.Integer");
-  public final TypeRepresentation FLOAT = fromQualifiedName("Standard.Base.Data.Numbers.Float");
+  public static final String numberQualifiedName = "Standard.Base.Data.Numbers.Number";
+  private static final String integerQualifiedName = "Standard.Base.Data.Numbers.Integer";
+  private static final String floatQualifiedName = "Standard.Base.Data.Numbers.Float";
+  public static final TypeRepresentation TEXT = fromQualifiedName("Standard.Base.Data.Text.Text");
+  public static final TypeRepresentation BOOLEAN =
+      fromQualifiedName("Standard.Base.Data.Boolean.Boolean");
+  public static final TypeRepresentation VECTOR =
+      fromQualifiedName("Standard.Base.Data.Vector.Vector");
+  public static final TypeRepresentation NOTHING =
+      fromQualifiedName("Standard.Base.Nothing.Nothing");
 
-  public final TypeRepresentation NUMBER = fromQualifiedName("Standard.Base.Data.Numbers.Number");
-  public final TypeRepresentation TEXT = fromQualifiedName("Standard.Base.Data.Text.Text");
-  public final TypeRepresentation VECTOR = fromQualifiedName("Standard.Base.Data.Vector.Vector");
-  public final TypeRepresentation NOTHING = fromQualifiedName("Standard.Base.Nothing.Nothing");
-
-  private TypeRepresentation fromQualifiedName(String qualifiedName) {
+  private static TypeRepresentation fromQualifiedName(String qualifiedName) {
     var fqn = QualifiedName$.MODULE$.fromString(qualifiedName);
     return new TypeRepresentation.AtomType(fqn, null);
   }
 
   public static boolean isAny(QualifiedName qualifiedName) {
     var str = qualifiedName.toString();
-    return str.equals("Standard.Base.Any.Any") || str.equals("Standard.Base.Any");
+    return str.equals(anyQualifiedName) || str.equals("Standard.Base.Any");
```

Comment:
Two names for `Any`? Isn't the first one the `Any` type and the second one the `Any.enso` module?

Reply:
Thanks for making me rethink that!

It used to be a hack that made something work. But it seems that it is no longer needed, so indeed it should have been removed.